### ISSUE TITLE
Modules TypeScript something I'm unsure error

### ISSUE THREAD

**Issue by justrealmilk** [moment/luxon#1274]
It was working prior to updating from 3.0.1 to 3.0.2
```
Module not found: Error: Can't resolve './impl/formatter' in 'G:\my_project\node_modules\luxon\src'
Did you mean 'formatter.js'?
BREAKING CHANGE: The request './impl/formatter' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').     
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
ERROR in ./node_modules/luxon/src/info.js 6:0-41
Module not found: Error: Can't resolve './impl/formatter' in 'G:\my_project\node_modules\luxon\src'
Did you mean 'formatter.js'?
BREAKING CHANGE: The request './impl/formatter' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').     
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

webpack compiled with 1 error
```

Unsure what's happening here

**Comment by DavidArmendariz** [comment]
Commenting to get subscribed as I have the same issue in a clean react project

**Comment by icambron** [comment]
This is fixed in 3.0.3
